server/src/cli.rs

```diff
@@ -173,6 +173,10 @@ pub struct EdgeArgs {
     #[clap(short, long, env, value_delimiter = ',')]
     pub tokens: Vec<String>,
 
+    /// Set a list of frontend tokens that Edge will always trust. These need to either match the Unleash token format, or they're an arbitrary string followed by an @ and then an environment, e.g. secret-123@development
+    #[clap(short, long, env, value_delimiter = ',')]
+    pub pretrusted_tokens: Option<Vec<String>>,
```

Comment:
Since you have a parser ready, using it through clap, so you either have successful tokens or none at all once you get to your main method?

Reply:
Yesss! That's a great idea, I'll patch this

Reply:
It does end up as a vec of tuples of Vec<(String, EdgeToken)> but I think that's okay. This means you now specify the token formats correctly or get an error message, which I actually think it better